Common/include/CConfig.hpp

```diff
@@ -1147,7 +1147,7 @@ class CConfig {
   unsigned long *VolumeOutputFrequencies; /*!< \brief list containing the writing frequencies */
 
   bool Multizone_Mesh;            /*!< \brief Determines if the mesh contains multiple zones. */
-  bool SinglezoneDriver;          /*!< \brief Determines if the single-zone driver is used. (TEMPORARY) */
+  bool SinglezoneDriver = true;   /*!< \brief Determines if the single-zone driver is used. (TEMPORARY) */
```

Comment:
Is this still temporary?

Reply:
Good point, the boolean can be removed and deduced from "not multizone" but some of the logic is still required.